src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -193,6 +193,12 @@ class CommandRequestImpl : public CommandImpl,
   bool CheckAllowedParameters();
 
   /**
+   * @brief Checks HMI capabilities for specified button support
+   * @param button Button to check
+   */
+  bool CheckHMICapabilities(mobile_apis::ButtonName::eType button) const;
```

Comment:
@AKalinich-Luxoft 
Perhaps it worth to add 'const' to parameter just for better style.